charts/gateway-operator/charts/kubernetes-configuration-crds/README.md

```diff
@@ -0,0 +1,13 @@
+# kubernetes-configuration subchart
+
+This sub-chart contains Kong's [Kubernetes Configuration][kconf] CRDs, allowing users to control whether to install them.
+
+[kconf]: https://github.com/Kong/kubernetes-configuration
+
+### Updating CRDs
+
+To update the CRDs, you can run the following command:
+
+```bash
+kustomize build github.com/kong/kubernetes-configuration/config/crd > ./charts/gateway-operator/charts/kubernetes-configuration/crds/kubernetes-configuration-crds.yaml
+```
```

Comment:
We should explicitly call this out that this is not for updating the CRDs in one's cluster (user facing guide) but for developers maintaining this repo to update the CRD definitions in this repo.